extensions/spring-data-jpa/runtime/src/main/java/io/quarkus/spring/data/runtime/TypesConverter.java

```diff
@@ -35,6 +35,11 @@ private TypesConverter() {
     }
 
     public static io.quarkus.panache.common.Page toPanachePage(org.springframework.data.domain.Pageable pageable) {
-        return new io.quarkus.panache.common.Page(pageable.getPageNumber(), pageable.getPageSize());
+        if (pageable.isPaged()) {
+            int pageNumber = pageable.getPageNumber();
+            int pageSize = pageable.getPageSize();
+            return new io.quarkus.panache.common.Page(pageNumber, pageSize);
+        }
+        return null;
```

Comment:
How come this works? How is `null` handled?

Reply:
I looked into it, and `null` works because https://github.com/quarkusio/quarkus/blob/master/extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/runtime/PanacheQueryImpl.java#L176 properly handles `null`. So let's just add a comment.